src/libraries/Native/Unix/System.Globalization.Native/pal_collation.c

```diff
@@ -112,6 +112,14 @@ static const UChar g_HalfFullHigherChars[] = {
 };
 static const int32_t g_HalfFullCharsLength = (sizeof(g_HalfFullHigherChars) / sizeof(UChar));
 
+static const UChar g_HiraganaWithoutDakutenChars[] = {
```

Comment:
>g_HiraganaWithoutDakutenChars [](http://example.com/codeflow?start=19&length=29)

nit: please add some comment here telling why we specicial case this range and how it will be used.